upup/pkg/fi/vfs_castore.go

```diff
@@ -59,7 +60,7 @@ func NewVFSCAStore(cluster *kops.Cluster, basedir vfs.Path) *VFSCAStore {
 }
 
 // NewVFSSSHCredentialStore creates a SSHCredentialStore backed by VFS
-func NewVFSSSHCredentialStore(cluster *kops.Cluster, basedir vfs.Path) SSHCredentialStore {
+func NewVFSSSHCredentialStore(ctx context.Context, cluster *kops.Cluster, basedir vfs.Path) SSHCredentialStore {
```

Comment:
Why is this changing to take a Context? We're adding a Context to all methods, so why one when creating the store?

Reply:
You're right - we don't need it - removed!